modules/Sources/Dependencies/AddressBookClient/AddressBookEncryption.swift

```diff
@@ -146,34 +118,34 @@ extension AddressBookClient {
     ///     [Unencrypted data]        `address book version`
     ///     [Unencrypted data]        `timestamp`
     ///     [Unencrypted data]        `contacts`
-    static func contactsFrom(unencryptedData: Data) throws -> AddressBookContacts {
+    static func contactsFrom(_ data: Data) throws -> AddressBookContacts {
         var offset = 0
         
         // Deserialize `version`
-        let versionBytes = unencryptedData.subdata(in: offset..<(offset + MemoryLayout<Int>.size))
-        offset += MemoryLayout<Int>.size
+        let versionBytes = try AddressBookClient.subdata(of: data, in: offset..<(offset + Constants.int64Size))
+        offset += Constants.int64Size
         
         // Deserialize and check `address book version`
         guard let version = AddressBookClient.bytesToInt(Array(versionBytes)), version == AddressBookContacts.Constants.version else {
             return .empty
         }
         
         // Deserialize `lastUpdated`
-        guard let lastUpdated = AddressBookClient.deserializeDate(from: unencryptedData, at: &offset) else {
+        guard let lastUpdated = try AddressBookClient.deserializeDate(from: data, at: &offset) else {
             return .empty
         }
         
         // Deserialize `contactsCount`
-        let contactsCountBytes = unencryptedData.subdata(in: offset..<(offset + MemoryLayout<Int>.size))
-        offset += MemoryLayout<Int>.size
+        let contactsCountBytes = try AddressBookClient.subdata(of: data, in: offset..<(offset + Constants.int64Size))
+        offset += Constants.int64Size
         
         guard let contactsCount = AddressBookClient.bytesToInt(Array(contactsCountBytes)) else {
             return .empty
         }
         
         var contacts: [Contact] = []
         for _ in 0..<contactsCount {
-            if let contact = AddressBookClient.deserializeContact(from: unencryptedData, at: &offset) {
+            if let contact = try AddressBookClient.deserializeContact(from: data, at: &offset) {
                 contacts.append(contact)
             }
         }
```

Comment:
On line 155, pass `version` instead of `AddressBookContacts.Constants.version`. They are currently the same, but passing `version` will just work once this changes.

Reply:
Updated